config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -100,3 +146,50 @@ periodics:
           limits:
             cpu: "2"
             memory: "6Gi"
+
+  # This jobs runs e2e.test with a focus on tests for all alpha storage features on a kind cluster
+  - name: ci-kubernetes-e2e-storage-kind-alpha-features
+    interval: 12h
+    annotations:
+      testgrid-dashboards: sig-storage-kubernetes
+      testgrid-tab-name: kind-alpha-features
+      testgrid-alert-email: kubernetes-sig-storage-test-failures@googlegroups.com
+      description: Run storage tests for alpha features in a KIND cluster.
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    cluster: eks-prow-build-cluster
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20240111-cf1d81388e-master
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
```

Comment:
Do we need to do ``make`` the tests and ``kind build`` like some other tests? Or it comes later when we actually have some tests to run

Reply:
@sunnylovestiramisu the e2e-k8s script will execute `kind build` as well.